pkg/apis/ptp/v1/ptpoperatorconfig_types.go

```diff
@@ -24,6 +25,21 @@ type PtpOperatorConfigStatus struct {
 	// Add custom validation using kubebuilder tags: https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html
 }
 
+// PtpEventConfig defines the desired state of event framework
+// +k8s:openapi-gen=true
+type PtpEventConfig struct {
+	//EnableEventPublisher will deploy event proxy as a sidecar
+	EnableEventPublisher  bool `json:"enableEventPublisher,omitempty"`
+	//clock state to stay in holdover state in secs
+	HoldOverTimeout int64 `json:"holdOverTimeout,omitempty"`
+	//max offset in nano secs
+	MaxOffsetThreshold int64 `json:"maxOffsetThreshold,omitempty"`
+	//min offset in nano secs
+	MinOffsetThreshold int64 `json:"minOffsetThreshold,omitempty"`
+    // transportHost address for event messages
```

Comment:
same

Reply:
Also added default  value so the CRD's show the default value
I made this data available via CLI 

$oc get ptpoperatorconfig
```
NAME      EVENT ENABLED   HOLDOVER TIMEOUT   MAX THRESHOLD   MIN THRESHOLD   EVENT TRANSPORT HOST
default   false           5                  100             -100            
```